.github/workflows/upload-samples.yml

```diff
@@ -0,0 +1,54 @@
+#  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+#  WSO2 Inc. licenses this file to you under the Apache License,
+#  Version 2.0 (the "License"); you may not use this file except
+#  in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied. See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+
+# This upload sample applications to latest release assets.
+
+name: Upload Samples
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+          
+    - name: Zip Samples into Assets Directory
+      working-directory: samples
+      run: |
+        mkdir ../assets
+        for dir in */ ; do
+            zip -r "../assets/${dir%/}.zip" $dir
+        done
+
+    - name: Get Latest Release Tag
+      id: get-latest-release-tag
+      run: |
+        get_latest_release() {
+            curl --silent "https://api.github.com/repos/${{ github.repository }}/releases/latest" | grep -Po '"tag_name": "\K.*?(?=")'
+        }
+        echo "::set-output name=LATEST_RELEASE_TAG:: $(get_latest_release)"
+
+    - name: Upload Samples to Latest Release
+      uses: svenstaro/upload-release-action@c5cd3d711a186bb9da45dd49f8a7d3c61ad2f4d4
+      with:
+        repo_token: ${{ secrets.ASGARDIO_GITHUB_BOT_TOKEN }}
```

Comment:
Shall we use a token with a bare minimal scope as this is a third-party workflow? Official GitHub upload-release-assets workflow is no longer maintained and do not support re-uploading multiple assets.